app/classifier.py

```diff
@@ -21,43 +21,64 @@
 classify_bp = Blueprint('classify_bp', __name__)
 
 
-def loadVocab(vocab: str):
-    with open(vocab, 'r') as f:
-        return f.readlines()
-
-
 class Classifier:
     """ Classifier may classify a string sequence's topic probability vector.
 
         Parameters:
-            path_to_bert (str): the URL path to a BERT model e.g.
-                    http://tf.google.com/bert_uncased.
-            path_to_classifier (str): the local path to the dir
-                    containing a saved classifier model and its variables.
-            path_to_vocab (str): the local path to a vocab label file to
-                    be used in constructing human-readable topic classification
-                    output.
+            base_classifier_dir (str): the local path to the dir
+                    containing all saved classifier models and their instances.
     """
-
-    def __init__(self, path_to_bert, path_to_classifier, path_to_vocab: str):
-        self.logger = logging.getLogger('app.logger')
-        self.vocab = loadVocab(path_to_vocab)
-        self.classifier = path_to_classifier
-        self.embedder = embedder.Embedder(path_to_bert)
-        self.predictor = tf.contrib.predictor.from_saved_model(self.classifier)
-
-    def classify(self, seq: str) -> [(str, float)]:
+    def __init__(self,
+                 base_classifier_dir: str):
+
+        self.logger = logging.getLogger()
+        self.base_classifier_dir = base_classifier_dir
+
+    def _load_instance(
+            self, relative_path_to_model: str):
+        instances = os.listdir(relative_path_to_model)
```

Comment:
os.listdir(path.join(base_classifier_dir, relative_path_to_model)) ?

Or if not, the 'relative_' has no meaning, it's just 'path_to_model'.

Reply:
dropped the relative